TezosKit/Michelson/CustomMichelsonParameter.swift

```diff
@@ -0,0 +1,15 @@
+// Copyright Keefer Taylor, 2019.
+
+import Foundation
+
+/// A custom Michelson paramater.
```

Comment:
Technically this is taking some Michelson code and converting it into the JSON Micheline representation.

Reply:
Clarified comment. I also renamed this class to `AbstractMichelineParameter` which is a bit of a better name, IMO.